test/integration/volume/persistent_volumes_test.go

```diff
@@ -666,7 +667,7 @@ func TestPersistentVolumeMultiPVsPVCs(t *testing.T) {
 
 			select {
 			case <-stopCh:
-				break
+				break loop
```

Comment:
should this instead return from the go routine?

Reply:
Good idea!